### PR TITLE
Update fuzzer

### PR DESCRIPTION
**Pull request by eckter**

### CODE REVIEW

File: api/osrd_infra/views/pathfinding.py [hunk 1]
```diff
@@ -174,9 +174,9 @@ def compute_vmax(payload: PathPayload, track_map: Mapping[str, TrackSection]):
     for track_id, track in track_map.items():
         trees[track_id].addi(0, track["length"], -1)
         for speed_section in track["speed_sections"]:
-            assert speed_section["begin"] < speed_section["end"]
-            trees[track_id].chop(speed_section["begin"], speed_section["end"])
-            trees[track_id].addi(speed_section["begin"], speed_section["end"], speed_section["speed"])
+            if speed_section["begin"] > speed_section["end"]:
+                trees[track_id].chop(speed_section["begin"], speed_section["end"])
+                trees[track_id].addi(speed_section["begin"], speed_section["end"], speed_section["speed"])
```
Comment (multun) [inline api/osrd_infra/views/pathfinding.py]: this change makes no sense to me

Comment (eckter) [reply api/osrd_infra/views/pathfinding.py]: There are some speed sections with a length of 0, the assert sometimes failed when using the fuzzer on the new infra. Ideally it shouldn't be imported in the first place, but I didn't want to re-import a new infra just for this

Comment (multun) [reply api/osrd_infra/views/pathfinding.py]: the condition seem to be broken, the > sign only runs the conditions where begin is after end, that's just weird

Comment (eckter) [reply api/osrd_infra/views/pathfinding.py]: Ah good catch, that's a mistake. It prevented the crash only because the broken case is when it's equal

Comment (multun) [reply api/osrd_infra/views/pathfinding.py]: :fishing_pole_and_fish:

**Pull request merged**